keyboards/ergodash/rev2/keymaps/vishnu/keymap.c

```diff
@@ -0,0 +1,116 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _ADJUST 1
+#define _FKeys 2
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  ADJUST,
+	FKeys,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
These lines can be safely deleted as QMK defines these automatically in the core code (which is automatically included).

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/quantum/quantum_keycodes.h#L29-L31

```suggestion
```